job-runner/chain/traverser.go

```diff
@@ -360,15 +373,21 @@ func (t *traverser) runJobs() {
 func (t *traverser) sendJL(pJob proto.Job, err error) {
 	jLogger := t.logger.WithFields(log.Fields{"job_id": pJob.Id})
 	jl := proto.JobLog{
-		RequestId:  t.chain.RequestId(),
-		JobId:      pJob.Id,
-		Name:       pJob.Name,
-		Type:       pJob.Type,
-		StartedAt:  0, // zero because the job never ran
-		FinishedAt: 0,
-		State:      pJob.State,
-		Exit:       1,
-		Error:      err.Error(),
+		RequestId:   t.chain.RequestId(),
+		JobId:       pJob.Id,
+		Name:        pJob.Name,
+		Type:        pJob.Type,
+		Try:         1 + t.chain.SequenceRetryCount(pJob),
+		SequenceTry: 1 + t.chain.SequenceRetryCount(pJob),
```

Comment:
How can both `Try` and `SequenceTry` be the same value? Also, there was no `Try` previously (in diff).

Reply:
And `Error:      err.Error(),` was removed?